draft-ietf-quic-ack-frequency.md

```diff
@@ -382,15 +382,17 @@ Unreported Missing:
   Missing that have not yet been received.
 
 An endpoint that receives an ACK_FREQUENCY frame with a non-zero Reordering
-Threshold value SHOULD send an immediate ACK when the gap
-between the smallest Unreported Missing packet and the Largest Unacked is greater
-than or equal to the Reordering Threshold value. Sending this additional ACK will
-reset the max_ack_delay timer and Ack-Eliciting Threshold counter (as any ACK
-would do).
-
-When an ack-eliciting packet is received with a packet number less than
-Largest Acked, this still triggers an immediate acknowledgement in an effort to
-avoid the packet being spuriously declared lost.
+Threshold value SHOULD send an immediate ACK whenever it observes
+unacknowledged state that lies outside of its configured Reordering Threshold,
+i.e.:
+* when the difference between the smallest Unreported Missing packet and the
+  Largest Unacked packet becomes greater than or equal to the Reordering
+  Threshold value; or
+* when the endpoint receives an ack-eliciting packet whose packet number is
+  less than `Largest Acked - Reordering Threshold`.
```

Comment:
Shouldn't this be "less than or equal to"?

IIUC, if Largest Acked is 10 and Reordering Threshold is 3, then packet 7 is at danger of being declared lost. so we should ack 7 immediately.

Reply:
agreed. I think this should be less than or equal to. Consider the case where Reordering Threshold = 1, largest ack is 4, then we should ack on receiving packet 3, which is consistent with RFC 9000.